patches/PSP.cmake

```diff
@@ -1,34 +1,64 @@
-SET(CMAKE_SYSTEM_NAME Generic)
-SET(CMAKE_SYSTEM_VERSION 1)
-SET(CMAKE_CROSSCOMPILING TRUE)
```

Comment:
Are these 2 lines not required?

Reply:
Documentation says `CMAKE_CROSSCOMPILING` is set to TRUE anyway if you pass `-DCMAKE_TOOLCHAIN_FILE`:

```
If cmake(1) is invoked with the command line parameter -DCMAKE_TOOLCHAIN_FILE=path/to/file, the file will be loaded early to set values for the compilers. The CMAKE_CROSSCOMPILING variable is set to true when CMake is cross-compiling.
```
From https://cmake.org/cmake/help/v3.6/manual/cmake-toolchains.7.html

Reply:
And for the `CMAKE_SYSTEM_VERSION` - as we set system to "Generic", I see no need for that